Pipfile

```diff
@@ -1,15 +1,21 @@
 [[source]]
+
```

Comment:
There's loads of extra spaces in this file now. Did you edit it by hand?

Reply:
nope - this happened automatically when I installed the `titlecase` package